dashboards-reports/server/model/index.ts

```diff
@@ -187,16 +187,10 @@ export const triggerSchema = schema.object({
 });
 
 export const deliverySchema = schema.object({
-  delivery_type: schema.oneOf([
-    schema.literal(DELIVERY_TYPE.opensearchDashboardsUser),
-    schema.literal(DELIVERY_TYPE.channel),
-  ]),
-  delivery_params: schema.conditional(
-    schema.siblingRef('delivery_type'),
-    DELIVERY_TYPE.opensearchDashboardsUser,
-    opensearchDashboardsUserSchema,
-    channelSchema
-  ),
+  configIds: schema.maybe(schema.arrayOf(schema.string())),
```

Comment:
why `maybe()`, isn't this a required field?

Reply:
Yep, good catch. Will fix that